public_html/wp-content/themes/pattern-directory/css/components/_patterns-search.scss

```diff
@@ -0,0 +1,3 @@
+.patterns-search__container {
+	margin-top: 1.5rem;
```

Comment:
You should be able to just add this margin to `.context-bar`, and margin collapsing means it'll still be fine when there's a menu.

Reply:
Yep. I tend to lean towards margin on containers and not components. I don't know how many times I've needed to overwrite the margin-bottom on the[ Gutenberg base control](https://github.com/WordPress/gutenberg/blob/ece5560abd945279530bd58f80f74f06b904a293/packages/components/src/base-control/styles/base-control-styles.js#L18) already :)

Reply:
Technically we want the margin on the component though, since ContextBar will always need space above it (whether it's between the menu & context bar or just spacing above the context bar).

If you'd rather put space on the container, can you put space on `site-main`, or give all the wrappers a new class? Really I just don't want the search page to have a special new style, when this does apply everywhere.

Reply:
The easiest and cleanest way I could find apart from adding classes to the main container is to target the specific page: `.search .site-header`.

Adding it to the component is tricky for 2 reasons:
1. The mobile layouts differ and there would need context-specific adding/removing to get it to work which would then again be specific to the page, or specific to the lack of menu.
2. The component shrinks its height to 0 to account for the transition, adding margin disturbs the layout when it's not active

Also, adding it to the `site-header` for non-homepage also doesn't work because the mobile version of the category navigation requires that there is no margin-bottom on the `site-header`, but it is required for the search page. So you need to conditionally add the margin which is again, not ideal.